www/nginx/src/opnsense/mvc/app/models/OPNsense/Nginx/Nginx.xml

```diff
@@ -590,6 +586,9 @@
       <headers type="BooleanField">
         <Required>Y</Required>
       </headers>
+      <body type="BooleanField">
+        <Required>Y</Required>
```

Comment:
@fabianfrz I just recognized (after pulling this to a staging system with existing rules) that introducing a new "Required" field is not the best idea, as this prevents saving of any rule unless further intervention.
Is there any model update foreseen in OPNsense plugins? Or should this better be changed to Required=N?

Reply:
Just figured out how the migration works. As here a new mandatory field has no default value this prevents automatic migration of `BaseModel`.
Fix proposed in #2152